### PR TITLE
Fixes #812: Added privacy page and linked it to footer

### LINKED ISSUE #812: Adding "PRIVACY" Page and linking it in the Footer
**Current behavior:**
There is no page for privacy policy in the footer.

**Expected behavior:**
There should be a link to the "PRIVACY" page in the footer on the right hand side.

**Would you like to work on this issue?**
Yes.

### PR DESCRIPTION
**Pull request by marvin08**
#### Changes proposed in this pull request:

I have added the privacy policy page of susper and linked it to footer.

screenshot from 2017-11-20 18 53 43

screenshot from 2017-11-20 18 53 05

Fixes #812.

### CODE REVIEW

**Comment by marvin08** [comment]
@harshit98 @mariobehling I have removed the Google Play Services link as suggested. Please review this PR.

File: src/app/auto-complete/auto-complete.component.css [hunk 1]
```diff
@@ -6,7 +6,6 @@ a {
   font-family: Arial, sans-serif;
   font-size: 17px;
   margin-left: 2.4%;
-  display: block;
```
Comment (aashutoshrathi) [inline src/app/auto-complete/auto-complete.component.css]: Please add this line back, why are you reverting changes made in #818?

Comment (marvin08) [reply src/app/auto-complete/auto-complete.component.css]: @aashutoshrathi But I didn't edit the 'auto-complete.component.css'.

Comment (aashutoshrathi) [reply src/app/auto-complete/auto-complete.component.css]: Oh then rebase, please 😅

**Comment by harshit98** [comment]
@marvin08 Whenever someone reviews your PR and suggest you changes, avoid making a new PR of it. What I have observed is -> you close the PR again and do the suggested changes and open a new PR for it. That's not correct and comes in bad practices. Always do the changes in the current PR.

**Comment by marvin08** [comment]
@harshit98 It did this because I faced some issues while squashing the commits. But in future I will keep this in mind :+1:.

**Comment by marvin08** [comment]
@aashutoshrathi @harshit98 @mariobehling I think now my branch is updated and this PR looks fine to me. So, please review this PR.

**Comment by marvin08** [comment]
Ok @bhaveshAn.

**Pull request merged**